dist/sharect.babel.js

```diff
@@ -103,16 +125,25 @@ var Sharect = function () {
 
       div.appendChild(arrow);
 
-      document.body.appendChild(div);
+      if (hasSelection()) {
```

Comment:
Appen tooltip to body or to nearest offsetParent() of selection; that makes the positioning simpler and with no need of adjusting it on scroll